README.md

```diff
@@ -736,6 +742,154 @@ No errors detected
 
 ```
 
+## `mcci-catena-provision-sigfox.py`
+
+This script communicates with catena to get information for register it in sigfox network via sigfox API, then it loads the catena script to device for complete the provisioning process.
+
+### Required
+
+* Python 3.* (Installation steps [here](https://realpython.com/installing-python/))
+* Install Python packages [pyserial](https://pyserial.readthedocs.io/en/latest/pyserial.html#installation), [requests](https://requests.readthedocs.io/en/master/) and [ruamel.yaml](https://yaml.readthedocs.io/en/latest/) using following commands in terminal/command prompt:
+  1. `pip install pyserial`

```

Comment:
We should recommend `pip3` (not `pip`, in case they have python 2 installed); and we should recommend that they do the update process for `pip3` before starting the installs.